docker/Dockerfile.cross

```diff
@@ -7,4 +7,4 @@ ARG checksum="d9d9b964d428733e98c2579b91772af4  /usr/bin/docker"
 RUN wget -O- https://download.docker.com/linux/static/stable/x86_64/docker-20.10.9.tgz | tar -C /usr/bin/ --strip-components=1 -xzf - docker/docker
 RUN echo $checksum | md5sum -c
 
-RUN cargo install cross
+RUN cargo install --git https://github.com/cross-rs/cross
```

Comment:
This is still wrong imo, I'd expect this to be built in our CI and published as `ghcr.io/cross-rs/cross-in-docker`.

Reply:
Yes, this needs to be something like

```
COPY . .
RUN cargo install --path . --locked
```